client/js/util.js

```diff
@@ -36,11 +36,55 @@ const featureTest = (property, value, noPrefixes) => {
   return mStyle[ property ].indexOf(value) !== -1;
 };
 
+const addClassesToElements = (elements, className) => {
```

Comment:
Why not embed fastDom here so we don't have to worry about it in the implementation?